### PR TITLE
Update documentation definitions for operators

### LINKED ISSUE #2015: Improve documentation content
### What should be done?

Make changes to syntax and improve explanations on operators.

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #2015.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/reference/operators/comparison-operators.md [hunk 1]
```diff
@@ -281,12 +286,12 @@ The output:
 
 ## $in
 
-*Syntax*: `{ field: { $in: [<value1>, <value2>, ... <valueN> ] } }`
+*Syntax*: `{ field: { $in: [<array-value1>, <array-value2>, ... <array-valueN> ] } }`
```
Comment (rumyantseva) [inline website/docs/reference/operators/comparison-operators.md:289]: Maybe `element1`, `element2` etc -- just like you did in `$all`?

File: website/docs/reference/operators/element-operators.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ db.electronics.insertMany([
 
 ## $exists
 
-*Syntax*: `{ <field>: { $exists: <boolean> } }`
+*Syntax*: `{ <field>: { $exists: <boolean-value> } }`
```
Comment (rumyantseva) [inline website/docs/reference/operators/element-operators.md:108]:
```suggestion
*Syntax*: `{ <field>: { $exists: <value> } }`
```

I think just value is enough as we explain it later

**Review by rumyantseva — DISMISSED** [review]
LGTM overall.

Some things look a bit inconsistent, e.g. in some places array values are `<array-value1>` and in the other place it's `<element1>`, but I think it's not super critical.

**Review by ptrfarkas — DISMISSED** [review]

File: website/docs/reference/operators/array-operators.md [hunk 3]
```diff
@@ -65,11 +65,15 @@ db.team.insertMany([
 
 ## $all
 
-*Syntax*: `{ <field>: { $all: [ <value1>, <value2>, ... ] } }`
+*Syntax*: `{ <field>: { $all: [ <element1>, <element2>, ... <elementN> ] } }`
 
-The `$all` operator matches documents where the field value is an array that contains all the specified elements.
+The `$all` operator matches documents where a particular field is an array containing all the elements given in the query.
```
Comment (chilagrow) [inline website/docs/reference/operators/array-operators.md:70]: `$all` operator selects documents with an array field that contains all elements from the given query.

Some suggestion, not exact wording.

File: website/docs/reference/operators/comparison-operators.md [hunk 4]
```diff
@@ -128,11 +131,11 @@ The response returns a single document that matches the query:
 
 ## $gt
 
-*Syntax*: `{ field: { $gt: value } }`
+*Syntax*: `{ field: { $gt: <query-value> } }`
```
[the remark this answers is not in the document]
Comment (Fashander) [reply website/docs/reference/operators/comparison-operators.md]: PTAL

**Review by ptrfarkas — DISMISSED** [review]

File: website/docs/reference/operators/array-operators.md [hunk 5]
```diff
@@ -65,11 +65,15 @@ db.team.insertMany([
 
 ## $all
 
-*Syntax*: `{ <field>: { $all: [ <value1>, <value2>, ... ] } }`
+*Syntax*: `{ <field>: { $all: [ <element1>, <element2>, ... <elementN> ] } }`
 
-The `$all` operator matches documents where the field value is an array that contains all the specified elements.
+Use the `all` operator when you want to select documents that contain every single element in a specified array.
```
Comment (chilagrow) [inline website/docs/reference/operators/array-operators.md:70]:
```suggestion
Use the `$all` operator when you want to select documents that contain every single element in a specified array.
```

File: website/docs/reference/operators/array-operators.md [hunk 6]
```diff
@@ -97,11 +101,12 @@ The output:
 
 ## $elemMatch
 
-*Syntax*: `{ <field>: { $elemMatch: { <query1>, <query2>, ... } } }`
+*Syntax*: `{ <field>: { $elemMatch: { <condition1>, <condition2>, ... <conditionN>} } }`
 
-The `$elemMatch` operator matches documents where the field value is an array that contains at least one element that matches the given condition.
+To select documents in a specified array field that matches at least one of several listed query conditions, use the `$elemMatch` operator.
```
Comment (chilagrow) [inline website/docs/reference/operators/array-operators.md:106]:
```suggestion
To select documents in a specified array field where one or more elements match all listed query conditions, use the `$elemMatch` operator.
```

Trying to highlight that we want to find one element or more to match all conditions.

File: website/docs/reference/operators/comparison-operators.md [hunk 7]
```diff
@@ -251,10 +257,12 @@ The output:
 
 ## $lte
 
-*Syntax*: `{ field: { $lte: value } }`
+*Syntax*: `{ field: { $lte: <element> } }`
 
-The less than or equal operator `$lte` selects document with values that are less than or equal to the specified query.
-The following operation queries for documents where the field `age` is less than or equal to `21`.
+The `lte` operator is the opposite of the `gte` operator.
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:262]:
```suggestion
The `$lte` operator is the opposite of the `$gte` operator.
```

File: website/docs/reference/operators/comparison-operators.md [hunk 8]
```diff
@@ -321,10 +328,13 @@ The output:
 
 ## $ne
 
-*Syntax*: `{ field: { $ne: value } }`
+*Syntax*: `{ field: { $ne: <element> } }`
+
+When selecting documents that do not match a specified query, use the `$ne` operator.
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:333]:
```suggestion
```

I would delete this sentence, the below sentence describe it better :hugs:

File: website/docs/reference/operators/comparison-operators.md [hunk 9]
```diff
@@ -72,10 +73,13 @@ db.employees.insertMany([
 
 ## $eq
 
-*Syntax*: `{ field: { $eq: value } }`
+*Syntax*: `{ field: { $eq: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:76]:
```suggestion
*Syntax*: `{ <field>: { $eq: <element> } }`
```

for consistency :hugs:

File: website/docs/reference/operators/comparison-operators.md [hunk 10]
```diff
@@ -128,11 +132,11 @@ The response returns a single document that matches the query:
 
 ## $gt
 
-*Syntax*: `{ field: { $gt: value } }`
+*Syntax*: `{ field: { $gt: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:135]:
```suggestion
*Syntax*: `{ <field>: { $gt: <element> } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 11]
```diff
@@ -167,10 +171,11 @@ The output:
 
 ## $gte
 
-*Syntax*: `{ field: { $gte: value } }`
+*Syntax*: `{ field: { $gte: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:174]:
```suggestion
*Syntax*: `{ <field>: { $gte: <element> } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 12]
```diff
@@ -213,10 +218,11 @@ The output:
 
 ## $lt
 
-*Syntax*: `{ field: { $lt: value } }`
+*Syntax*: `{ field: { $lt: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:221]:
```suggestion
*Syntax*: `{ <field>: { $lt: <element> } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 13]
```diff
@@ -251,10 +257,12 @@ The output:
 
 ## $lte
 
-*Syntax*: `{ field: { $lte: value } }`
+*Syntax*: `{ field: { $lte: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:260]:
```suggestion
*Syntax*: `{ <field>: { $lte: <element> } }`
```

File: website/docs/reference/operators/logical-operators.md [hunk 14]
```diff
@@ -241,11 +241,11 @@ The output:
 
 ## $not
 
-*Syntax*: `{ field: { $not: { <expression> } } }`
+*Syntax*: `{ field: { $not: { <condition> } } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/logical-operators.md:244]:
```suggestion
*Syntax*: `{ <field>: { $not: { <condition> } } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 15]
```diff
@@ -359,10 +369,12 @@ The output:
 
 ## $nin
 
-*Syntax*: `{ field: { $nin: [ <value1>, <value2> ... <valueN> ] } }`
+*Syntax*: `{ field: { $nin: [ <element1>, <element2> ... <elementN> ] } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:372]:
```suggestion
*Syntax*: `{ <field>: { $nin: [ <element1>, <element2> ... <elementN> ] } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 16]
```diff
@@ -321,10 +328,13 @@ The output:
 
 ## $ne
 
-*Syntax*: `{ field: { $ne: value } }`
+*Syntax*: `{ field: { $ne: <element> } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:331]:
```suggestion
*Syntax*: `{ <field>: { $ne: <element> } }`
```

File: website/docs/reference/operators/comparison-operators.md [hunk 17]
```diff
@@ -281,12 +289,11 @@ The output:
 
 ## $in
 
-*Syntax*: `{ field: { $in: [<value1>, <value2>, ... <valueN> ] } }`
+*Syntax*: `{ field: { $in: [<element1>, <element2>, ... <elementN> ] } }`
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:292]:
```suggestion
*Syntax*: `{ <field>: { $in: [<element1>, <element2>, ... <elementN> ] } }`
```

**Review by chilagrow — COMMENTED** [review]
Minor comments to keep the consistency. great work!

**Review by chilagrow — APPROVED** [review]
Thanks for this!

**Review by w84thesun — COMMENTED** [review]
Nice one!

**Review by w84thesun — DISMISSED** [review]

**Review by ptrfarkas — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]

**Review by ptrfarkas — APPROVED** [review]

**Pull request merged**